### PR TITLE
ULTRA Update `docs/setup.md`

### PR DESCRIPTION
**Pull request by JenishPatel99**
Motive: Remove previous reference to SMARTS installation

### CODE REVIEW

**Review by christianjans — COMMENTED** [review]
I think we can also maybe get rid of https://github.com/huawei-noah/SMARTS/blob/be15eaf0765782ba33e240ed5a48a37b01ac6d12/ultra/docs/setup.md#L17-L18 because it seems like `black` is part of the installation requirements now: https://github.com/huawei-noah/SMARTS/blob/c5a16088152e840dbaa9afe302e8bb9338bc6a93/ultra/setup.py#L51

Also, were we going to add anything about the Sumo dependency?

**Comment by JenishPatel99** [comment]
> I think we can also maybe get rid of
> 
> https://github.com/huawei-noah/SMARTS/blob/be15eaf0765782ba33e240ed5a48a37b01ac6d12/ultra/docs/setup.md#L17-L18
> 
> because it seems like `black` is part of the installation requirements now:
> https://github.com/huawei-noah/SMARTS/blob/c5a16088152e840dbaa9afe302e8bb9338bc6a93/ultra/setup.py#L51
> 
> Also, were we going to add anything about the Sumo dependency?

Oh yeah thank you, I have removed the black installation step now and also have added the `install_deps.sh" to the base ultra directory.

File: ultra/docs/setup.md [hunk 1]
```diff
@@ -4,27 +4,26 @@ ULTRA can be run natively on your machine, or through a Docker container. See in
...
 # 1 - Navigate to the ULTRA directory.
 $ cd <path_to_SMARTS>/SMARTS/ultra
 
-# 2 - Create a virtual environment.
+# 2 - Install overhead dependencies (python3.7 & sumo)
+./install_deps.sh
+
+# 3 - verify sumo is >= 1.5.0
+# if you have issues see ./docs/SUMO_TROUBLESHOOTING.md
+sumo
```
Comment (christianjans) [inline ultra/docs/setup.md:16]:
```suggestion
$ ./install_deps.sh

# 3 - verify sumo is >= 1.5.0
# if you have issues see ./docs/SUMO_TROUBLESHOOTING.md
$ sumo
```
The other commands in this setup have a '$' preceding them. It definitely doesn't have to be there, but maybe we should just keep a consistent notation (either with or without the '$' for all commands).

Comment (JenishPatel99) [reply ultra/docs/setup.md]: No that is right, we should keep them consistent, nice catch!!

**Review by christianjans — APPROVED** [review]
Looks great!!

Now that we have an install script, maybe we should have a CI test that tries to install ULTRA's dependencies and run ULTRA on Linux and MacOS. Not in this PR of course, but maybe another thing to add later? What are your thoughts?

**Comment by JenishPatel99** [comment]
> Looks great!!
> 
> Now that we have an install script, maybe we should have a CI test that tries to install ULTRA's dependencies and run ULTRA on Linux and MacOS. Not in this PR of course, but maybe another thing to add later? What are your thoughts?

I was thinking about including the installation of those overhead dependencies from `install_deps.sh` in our `ci-ultra-tests.yml`. However, it seems like we are using a docker container (`huaweinoah/smarts:v0.4.13-minimal`) that probably comes with the python3.7 and sumo deps pre-installed. Are you suggesting to use a fresh Linux (Ubuntu)/MacOS container where we can install **all** (overhead & pip packages) dependencies and then run a basic ULTRA test?

**Comment by christianjans** [comment]
Yeah, that's what I had in mind. Like to use an Ubuntu 18.04 container for the Linux test and something like MacOS 10.15 for the Mac test. Looking at it again though, it doesn't look like SMARTS has a test like this, so maybe it's not necessary?

**Pull request merged**